.github/workflows/script/verify-pr-checks.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+# Sanity check that repo is clean to start with
+if [ ! -z "$(git status --porcelain)" ]; then
+    # If we get a fail here then this workflow needs attention...
+    >&2 echo "Failed: Repo should be clean before testing!"
+    exit 1
+fi
+
+# Wipe the generated-pr-checks directory incase there are extra unnecessary files in there
+rm -rf .github/generated-pr-checks
+
+# Generate the PR checks
+cd pr-checks && python3 sync.py
+
+# Check that repo is still clean
+if [ ! -z "$(git status --porcelain)" ]; then
+    # If we get a fail here then the PR needs attention
+    git diff
+    git status
+    >&2 echo "Failed: PR checks are not up to date. Run 'cd pr-checks && python3 sync.py' to update"
```

Comment:
I'm a little confused here. Wasn't `cd pr-checks && python3 sync.py` run above? Should the message be something like this:

```suggestion
    >&2 echo "Failed: PR checks are not up to date. Commit the changes and run again"
```

Reply:
Oh...I see this is going to be run on CI, but the command to update needs to be run locally.